test/infrastructure/inmemory/internal/server/api/handler.go

```diff
@@ -149,6 +149,14 @@ func (h *apiServerHandler) apisDiscovery(req *restful.Request, resp *restful.Res
 			}
 			return
 		}
+		if req.PathParameter("group") == "apps" && req.PathParameter("version") == "v1" {
```

Comment:
q: what if we simplify this func and return the value we get from getAPIResourceList in this func?